### PR TITLE
DEV: Upgrade Rails to version 7.1

### PR DESCRIPTION
**Pull request by Flink**

### CODE REVIEW

**Review by CvX — APPROVED** [review]

File: spec/fixtures/mmdb/mmdb.tar.gz
Comment (ZogStriP) [inline spec/fixtures/mmdb/mmdb.tar.gz:1]: What is this file for? 🤔

Comment (Flink) [reply spec/fixtures/mmdb/mmdb.tar.gz:1]: It’s actually a leftover, removing :grin: :+1:

Comment (ZogStriP) [reply spec/fixtures/mmdb/mmdb.tar.gz:1]: Oh, that's the MaxMind database. Why is it required now?

Comment (ZogStriP) [reply spec/fixtures/mmdb/mmdb.tar.gz:1]: Makes sense. I was curious if a change in Rails made it mandatory, but nope, that was us 😅

**Pull request merged**